lib/MultiColumnList/MCLRenderer.js

```diff
@@ -1155,11 +1174,24 @@ class MCLRenderer extends React.Component {
       let cellStyle = null;
       if (columnWidths[col]) {
         cellStyle = { width: this.state.columnWidths[col] };
+        if (colIndex === columns.length - 1 &&
+          get(columnWidthsProp, `${col}.max`) === undefined) {
+          cellStyle = { minWidth: this.state.columnWidths[col], flex: 'auto' };
+        }
       }
 
       const showOverflow = (columnOverflow[col]) ? css.mclShowOverflow : '';
       const colWidthsVar = columnWidths[col] || columnWidthsProp[col];
-      const shouldMeasure = (!columnWidths || !colWidthsVar);
+      // const shouldMeasure = (!columnWidths && !colWidthsVar);
+
+      let shouldMeasure = false;
+      if (modColumns.includes(col)) {
```

Comment:
I suggest simplifying a bit.

```suggestion
    const shouldMeasure = modColumns.includes(col) && !columnWidths[col] || !colWidthsVar;
```